packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js

```diff
@@ -167,6 +167,28 @@ QUnit.module('MessageGroup', moduleConfig, () => {
                 assert.deepEqual(avatar.option('url'), passedUrlValue);
             });
         });
+
+        QUnit.test('avatar component should be initialized with correct alt property', function(assert) {
+            [
```

Comment:
Minus of this approach is: you don't know what configuration is wrong, I suggest add comment  to assert

Reply:
I don't have any abjections to take into account this in future changes. This change is not critical.